config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml

```diff
@@ -194,6 +194,7 @@ periodics:
   cron: 0 8-20/12 * * *
   labels:
     preset-e2e-scalability-common: "true"
+    preset-e2e-scalability-periodics: "true"
```

Comment:
Let's not merge change release branches in the same step - we should first to head-related and only then backport.

Reply:
Done.